packages/runtime-handler/package.json

```diff
@@ -73,7 +73,10 @@
     "nocache": "^2.1.0",
     "normalize.css": "^8.0.1",
     "serialize-error": "^7.0.1",
-    "twilio": "3.29.2"
+    "twilio": "3.80.0"
```

Comment:
Late comment here, but shouldn't this match the default in Functions? And then we have the method that requires the project's Twilio dependency and falls back to this one? @dkundel